nucypher/blockchain/eth/domains.py

```diff
@@ -0,0 +1,70 @@
+from enum import Enum
+from typing import NamedTuple
+
+
+class ChainInfo(NamedTuple):
+    id: int
+    name: str
+
+
+class EthChain(ChainInfo, Enum):
+    MAINNET = (1, "mainnet")
+    GOERLI = (5, "goerli")
+    SEPOLIA = (11155111, "sepolia")
+    TESTERCHAIN = (131277322940537, "eth-tester")
+
+
+class PolygonChain(ChainInfo, Enum):
+    POLYGON = (137, "polygon")
```

Comment:
Since you're already in a "polygon" namespace, you can further disambiguate and just call this "mainnet" (`PolygonChain.MAINNET`)

```suggestion
    MAINNET = (137, "polygon")
```

Reply:
✔️